src/main/kotlin/me/partlysanestudios/partlysaneskies/config/OneConfigScreen.kt

```diff
@@ -726,22 +726,54 @@ object OneConfigScreen : Config(
     )
     var healerAlertCooldownSlider = 3.5f
 
-    // Pearl Refill
+    // Item Refill
     @Switch(
-        name = "Auto Pearl Refill",
-        description = "Automatically refills your pearls when a run starts.",
+        name = "Auto Item Refill",
+        description = "Automatically refills your utility items when a run starts.",
         category = "Dungeons",
-        subcategory = "Pearl Refill"
+        subcategory = "Item Refill"
     )
     var autoPearlRefill = false
```

Comment:
That resets config

Reply:
Only that one value, and since you basically changed the whole feature + auto enabled other items to refill, I'm sure it's better if disabled at first

Reply:
Fixed